dref/serializers.py

```diff
@@ -66,6 +67,8 @@ class Meta:
         read_only_fields = ("created_by",)
 
     def create(self, validated_data):
+        if "file" in validated_data:
+            check(validated_data["file"])
```

Comment:
For create, We can ignore "file" in validated_data check as the `file is a required field in DrefFile.

To make sure it is validated for update as well, a function validate_ will be easier.

```python
def validate_file(self, file):
    check(file)
    return file
```